public/static/docs/command-reference/diff.md

```diff
@@ -1,41 +1,53 @@
 # diff
 
-Show changes between commits in the <abbr>DVC repository</abbr>, or between a
-commit and the <abbr>workspace</abbr>. The comparison can be narrowed down to
-specific target files/directories tracked by DVC.
+Show added, modified, or deleted DVC-tracked files and directories between
+commits in the <abbr>DVC repository</abbr>, or between a commit and the
+workspace.
 
 ## Synopsis
 
 ```usage
-usage: dvc diff [-h] [-q | -v] [-t TARGET] a_ref [b_ref]
+usage: dvc diff [-h] [-q | -v]
+                [--show-json] [--show-hash]
+                [a_rev] [b_rev]
 
 positional arguments:
-  a_rev                 Old Git commit to compare (defaults to HEAD)
-  b_rev                 New Git commit to compare (defaults to the
-                        current workspace)
+  a_rev      Old Git commit to compare (defaults to HEAD)
+  b_rev      New Git commit to compare (defaults to the current workspace)
```

Comment:
> Per https://github.com/iterative/dvc.org/pull/953#discussion_r383098998

Not "old" or "new". Just "A" and "Another" or something like that...

- [ ] I can send a PR to the core repo with whatever you come up with here.

Reply:
let's create a ticket in the core repo and update the document when the ticket is done

Reply:
(I just don't see this as a big issue worth spending time now tbh)

Reply:
Well, there's already a lot of diff issues in that repo. I think I'll just submit a draft PR directly instead and see what the core team thinks...